### ISSUE TITLE
Workplane.save()

### ISSUE THREAD

**Issue by roipoussiere** [CadQuery/cadquery#1153]
I like the simplicity of [`Assembly.save()`](https://cadquery.readthedocs.io/en/latest/classreference.html#cadquery.Assembly.save):

```py
asm = ( cq.Assembly()
    .add(box)
    .save('box.step')
)
```

To export a model, in the other hand, the syntax is not as nice:

```py
box = cq.WorkPlane().box(1, 2, 3)
cq.exporters.export(box, 'box.step')
```

I wonder if the assembly export syntax could be applied on a model as well, for consistency and simplicity:

```py
box = ( cq.WorkPlane()
    .box(1, 2, 3)
    .save('box.step')
)
```

**Comment by adam-urbanczyk** [comment]
It would be nice indeed. @jmwright @lorenzncode would you agree?

**Comment by lorenzncode** [comment]
Yes, that might be convenient and I think it could support saving to BREP file too!

**Comment by jmwright** [comment]
+1

**Comment by adam-urbanczyk** [comment]
Workplane, Sketch, Shape and Assembly support `export`.
